examples/gno.land/p/demo/auction/auction.gno

```diff
@@ -0,0 +1,140 @@
+package auction
+
+import (
+	"std"
+	"strconv"
+	"time"
+
+	"gno.land/p/demo/avl"
+)
+
+// Main struct
```

Comment:
There is no needs for comments like these :)

Reply:
I'm sorry for the delay in replying :pray:.I deleted the comment :sweat_smile: ,and replaced avlTree with a simple slice  [6936b8](https://github.com/gnolang/gno/pull/2265/commits/6936b8b378e3cb753dda557445df4a9481805547)